BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -486,6 +497,26 @@ + (NSString *)APIBaseURL
     return staticAPIBaseURL;
 }
 
+//NOTE: Right now we assume that version-portion of the URL string is specified by the last component in the URL path and it also starts with a number
++ (NSString *)APIBaseURLWithoutVersion
+{
+    if (staticAPIBaseURLWithoutVersion.length == 0) {
+        NSString *versionedURL = [BOXContentClient APIBaseURL];
+        NSString *unversionedURL = nil;
+        NSURL *url = [NSURL URLWithString:versionedURL];
```

Comment:
so, we can skip these 3 lines and just use the NSString lastPathComponent, right?